File: CS/PrintDXGridSample/MainWindowViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
// ...

namespace PrintDXGridSample {
    public class MainWindowViewModel {
        IEnumerable<Person> persons;

        public IEnumerable<Person> Persons {
            get {
                return persons;
            }
            set {
                persons = value;
            }
        }

        public MainWindowViewModel() {
            persons = new List<Person>(new Person[] { 
                new Person("John", 25),
                new Person("Mary", 33)
            });
        }
    }
}

File: VB/PrintDXGridSample/MainWindowViewModel.vb
Imports System.Collections.Generic
' ...

Namespace PrintDXGridSample
    Public Class MainWindowViewModel
        Public Property Persons() As IEnumerable(Of Person)

        Public Sub New()
            Persons = { _
                New Person("John", 25), _
                New Person("Mary", 33) _
            }
        End Sub
    End Class
End Namespace
